.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ develop ]
   pull_request:
-    branches: [ develop, master, increment/*, release/* ]
+    branches: [ develop, master, increment/*, release/*, feature/* ]
```

Comment:
Will be removed if needed

Reply:
Make remove it!

Reply:
resolved in [90452dc](https://github.com/BlackBeltTechnology/judo-meta-jsl/pull/71/commits/90452dc5df1bf9de03a9e0cc2096619e5afaae3a)